src/pages/MapPage.js

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { MapCenterContext } from '../context/MapCenterContext';
+import { Map } from '../components/Map';
+import { LocationsContext } from '../context/LocationsContext';
+import Spinner from '../components/Spinner';
+import ErrorMessage from '../components/ErrorMessage';
+
+const MapPage = () => {
+  const valueCenterMap = useContext(MapCenterContext);
+  const valueLocations = useContext(LocationsContext);
+  const { defaultCenterMap, newCenterMap } = valueCenterMap;
+  const { locations, status, error } = valueLocations;
+
+  return (
+    <>
+      <h1>What's Near Me?</h1>
+      <div className="view__content">
+        {status === 'loading' && <Spinner />}
+        {status === 'error' && <ErrorMessage error={error} />}
+        {(status === 'success' || status === 'cancelled') && (
```

Comment:
If you wanted to keep the map showing on drag, the logic would need to change here somehow.

Reply:
I just write a comment on our Slack channel, and I put your question.  Now, we need to wait until that we have some answers. Thank you!

Reply:
That shouldn't hold up this PR, though. You can go ahead and track that work in a separate ticket if that's what everyone decides.

Reply:
Thank you for your clarification! We will merge & close this PR because we have already two approvals ✔️✔️.